docs/frameworks/symfony.md

```diff
@@ -105,47 +127,39 @@ The application is now ready to be deployed. Follow [the deployment guide](/docs
 
 ## Console
 
-As you may have noticed, we define a function of type "console" in `serverless.yml`. That function is using the [Console runtime](/docs/runtimes/console.md), which lets us run the Symfony Console on AWS Lambda.
+As you may have noticed, we define a function of type "console" in `serverless.yml`. That function is using the [Console runtime](/docs/runtimes/console.md),
+which lets us run the Symfony Console on AWS Lambda.
 
 To use it follow [the "Console" guide](/docs/runtimes/console.md).
 
 ## Logs
 
-While overriding the log's location in the `Kernel` class was necessary for Symfony to run correctly, by default Symfony logs in `stderr`. That is great because Bref [automatically forwards `stderr` to AWS CloudWatch](/docs/environment/logs.md).
+While overriding the log's location in the `Kernel` class was necessary for Symfony to run correctly, by default Symfony logs in `stderr`. That is great because
```

Comment:
We don't need to say this as it is internal to the Bridge

```suggestion
By default Symfony logs in `stderr`. That is great because
```

Reply:
Indeed, I changed it